### ISSUE TITLE
connecting to x11vnc/libvncserver w/o websockify doesn't work

### ISSUE THREAD

**Issue by thelamer** [novnc/noVNC#535]
Copied title from #262 

Using latest head a0e7ab43dca0ce11a713694ee4cf530bd3b17c5a for noVNC and downloaded and compiled the latest x11vnc from here http://sourceforge.net/projects/libvncserver/files/x11vnc/0.9.13/ . 

**Output from x11vnc on connection attempt:**

```
01/10/2015 08:38:36 Got connection from client 192.168.1.67
01/10/2015 08:38:36   other clients:
01/10/2015 08:38:36 Disabled X server key autorepeat.
01/10/2015 08:38:36   to force back on run: 'xset r on' (3 times)
01/10/2015 08:38:36 incr accepted_client=1 for 192.168.1.67:51038  sock=11
01/10/2015 08:38:36 rfbProcessClientProtocolVersion: not a valid RFB client: GET /websock
01/10/2015 08:38:36 client_count: 0
01/10/2015 08:38:36 Restored X server key autorepeat to: 1
01/10/2015 08:38:36 Client 192.168.1.67 gone
01/10/2015 08:38:36 Statistics             events    Transmit/ RawEquiv ( saved)
01/10/2015 08:38:36  TOTALS              :      0 |         0/        0 (  0.0%)
01/10/2015 08:38:36 Statistics             events    Received/ RawEquiv ( saved)
01/10/2015 08:38:36  TOTALS              :      0 |         0/        0 (  0.0%)
```

**Javascript error:**

```
WebSocket connection to 'ws://192.168.1.67:5900/websockify' failed: Error during WebSocket handshake: Invalid status line
```

**To duplicate:(ubuntu 14.04)**

```
apt-get update
apt-get install build-essential libxtst-dev libjpeg-dev libssl-dev xvfb x11-apps -y
wget http://downloads.sourceforge.net/project/libvncserver/x11vnc/0.9.13/x11vnc-0.9.13.tar.gz
tar -xzf x11vnc-0.9.13.tar.gz
cd x11vnc-0.9.13
./configure
make
cd x11vnc
Xvfb :0 -screen 0 640x480x16 &
DISPLAY=:0.0 xeyes &
./x11vnc -display :0.0 -rfbport 5900 -forever
```

Then just attempt to connect from the noVNC client. 

Please let me know if you need anymore information.

**Comment by DirectXMan12** [comment]
Hmm... I just tried reproducing this with the latest release of x11vnc, and it seems to work fine.  Please re-open this issue if you are still having problems.  If websockets were configured when you compiled x11vnc, you should see `Got 'ws' WebSockets handshake` in your x11vnc output.

**Comment by veselins** [comment]
@DirectXMan12 I have tested what you have said. And it's not working for me neither. 
I have downloaded:
`wget http://downloads.sourceforge.net/project/libvncserver/x11vnc/0.9.13/x11vnc-0.9.13.tar.gz`

configured it with:
`cd x11vnc-0.9.13
./configure --with-websockets
make
cd x11vnc
./x11vnc`

and still has the same error as @thelamer has mentioned:
`18/12/2016 10:29:56 rfbProcessClientProtocolVersion: not a valid RFB client: GET /websock`

One more thing, between noVNC and x11vnc I am using nginx reverse websocket proxy. Which configuration works perfectly when we have:
noVNC->reverse websocket proxy->websockify->x11vnc

@DirectXMan12 any advise? May you provide list of commands in the way you have installed x11vnc and way you use noVNC.?
